docs/_data/portland-2021-schedule.yaml

```diff
@@ -0,0 +1,73 @@
+writing_day:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:30'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '9:45'
+    title: Team up and start working
+  - time: '12:30'
+    title: Lunch Break
+  - time: '5:00'
+    title: End of Writing Day
+
+talks_day1:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:00'
+    title: Write the Docs Team - Introduction to Write the Docs
+  - time: '9:45'
+    title: "<b>Unconference Starts</b>"
+  - time: '9:45'
+    slug: writing-a-perfect-technical-tutorial-jessica-garson
+  - time: '10:30'
+    slug: shuffle-ball-change-sashay-your-way-to-clearer-api-documentation-rachael-stavchansky
+  - time: '11:15'
+    slug: a-guide-to-getting-started-in-open-source-abigail-mccarthy
+  - time: '12:00'
+    slug: beyond-metrics-using-maturity-models-to-develop-a-docs-strategy-sarah-r-rodlund
+  - time: '12:45'
+    title: Lunch (60 mins)
+  - time: '1:45'
+    title: Lightning Talks (30 mins)
+  - time: '2:15'
+    slug: the-secret-history-of-libraries-paris-buttfield-addison
+  - time: '3:00'
+    slug: putting-the-tech-in-technical-writer-swapnil-ogale
+  - time: '3:45'
+    slug: always-complete-never-finished-daniele-procida
+  - time: '4:30'
+    title: Day 1 wraps up
+  - time: '5:00'
+    title: '<b>Social Event</b>'
+  - time: '7:00'
+    title: '<b>Social Event Ends</b>'
+
+talks_day2:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:00'
+    slug: documentation-communities-sound-strategy-or-documentarian-s-gambit-laura-novich
+  - time: '9:45'
+    title: "<b>Job Fair Starts</b>"
+  - time: '9:45'
+    slug: building-a-style-guide-from-the-ground-up-lessons-learned-from-a-lone-writer-deanna-thompson
+  - time: '10:30'
+    slug: almost-none-to-some-driving-disqo-s-doc-culture-as-a-solo-documentarian-falon-darville
+  - time: '11:15'
+    slug: is-tech-writer-a-tester-and-vice-versa-is-tester-a-tech-writer-ines-stefanovic
+  - time: '12:00'
+    slug: writing-documentation-with-neurodivergent-open-source-contributors-in-mind-rin-oliver
+  - time: '12:45'
+    title: Lunch (60 mins)
+  - time: '1:45'
+    title: Lightning Talks (30 mins)
+  - time: '2:15'
+    title: "<b>Unconference Starts</b>"
```

Comment:
This feels weak. If we only have 2 afternoon sessions, we should probably do something different w/ the unconf. Not 100% sure what tho.

I'd propose we move lunch forward to 12, which would balance things more nicely.